articles/aks/keda-deploy-add-on-cli.md

```diff
@@ -0,0 +1,537 @@
+---
+title: Deploy the Kubernetes Event-driven Autoscaling (KEDA) add-on by using Azure CLI
+description: Use Azure CLI to deploy the Kubernetes Event-driven Autoscaling (KEDA) add-on to Azure Kubernetes Service (AKS).
+author: Ramya Oruganti
+ms.author: raorugan
+ms.service: container-service
+ms.topic: how-to article
+ms.date: 06/08/2022
+ms.custom: template-how-to 
+---
+
+
+# Install the Kubernetes Event-driven Autoscaling (KEDA) add-on by using Azure CLI
+
+This article shows you how to install the Kubernetes Event-driven Autoscaling (KEDA) add-on to Azure Kubernetes Service (AKS) by using Azure CLI. The article includes steps to verify that it's installed and running
+
+[!INCLUDE [Current version callout](./includes/keda/current-version-callout.md)]
+
+[!INCLUDE [preview features callout](./includes/preview/preview-callout.md)]
+
+
+## Prerequisites
+
+- An Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/free).
+- [Azure CLI installed](/cli/azure/install-azure-cli).
```

Comment:
added guidance on aks extension to be upgraded if the extension is incompatible with az cli version it will throw error and user will need to upgrade the az cli appropriately

Reply:
Wouldn't it be good to also simply list what version is required? This would be more customer-friendly